resources/views/docs/_structures/group.md

```diff
@@ -5,7 +5,6 @@ This object is not returned by any endpoints yet. It is here only as a reference
 Field           | Type    | Description
 ----------------|---------|------------------------------------------------------------
 colour          | string? | |
-description     | string? | |
```

Comment:
Where's the updated doc for this

Reply:
well there's no way to access it from api anymore, since there is no group endpoint and it's not included to usergroup